homeassistant/components/tessie/__init__.py

```diff
@@ -0,0 +1,69 @@
+"""Tessie integration."""
+from datetime import timedelta
+from http import HTTPStatus
+import logging
+
+from aiohttp import ClientConnectionError, ClientResponseError
+from tessie_api import get_state_of_all_vehicles
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_ACCESS_TOKEN, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import DOMAIN
+
+TESSIE_SYNC_INTERVAL = 15
+PLATFORMS = [Platform.SENSOR]
```

Comment:
While not required - I think it is typically better for these to be in const

Reply:
I checked a few other integrations and this always seems to be in init.